### ISSUE TITLE
require "Common" preventing game load

### ISSUE THREAD

**Issue by ChrisClark13** [Penguin-Spy/skins-factored#2]
looks like a capitalization error is keeping it from working

I've been using this mod with a private version of the Recolorable Protogen Character mod I changed to use your API.

Upon updating it to schema version 2, I get this error on game load.

```
Loading mod skins-factored 1.0.0 (data.lua)
Error ModManager.cpp:1608: Failed to load mod "skins-factored": __skins-factored__/data.lua:4: module Common not found;  no such file __skins-factored__/Common.lua no such file __skins-factored__/Common.lua no such file __core__/lualib/Common.lua
stack traceback:
	[C]: in function 'require'
	__skins-factored__/data.lua:4: in main chunk
```

**Comment by Penguin-Spy** [comment]
huh. on my computer this works just fine (although that still should be lowercase). do you by chance happen to use Linux/OSX (Mac)? i think `require` might work slightly differently due to the different file systems compared to Windows (what i regrettably use).
anyways, i need to fix some other stuff with that file & work on some more mod compat stuff; i'll probably have this fixed today or tomorrow.

**Comment by ChrisClark13** [comment]
changing the uppercase "C" to lowercase "c" fixed that file for me, so I'm good for now

**Comment by Penguin-Spy** [comment]
should be fixed in `v1.0.1` (uploaded to the mod portal, ignore the failed github workflow)
